src/cargo/core/compiler/layout.rs

```diff
@@ -47,32 +70,71 @@
 //!     # Hidden directory that holds all of the fingerprint files for all
 //!     # packages
 //!     .fingerprint/
+//!         # Each package is in a separate directory.
+//!         $pkgname-$META/
+//!             # Set of source filenames for this package.
+//!             dep-lib-$pkgname-$META
+//!             # Timestamp when this package was last used.
+//!             invoked.timestamp
```

Comment:
I love the new documentation! I think this is the start of package last built, not last used.

Reply:
Oops, thanks!